### PR TITLE
make the project a go module

### LINKED ISSUE #27: Make the project a Go module

### PR DESCRIPTION
**Pull request by justinmueller05082000**
fixes #27

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module example.com/trainee-sort
```
Comment (Al2Klimov) [inline go.mod]: Why "example.com"?

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (justinmueller05082000) [reply go.mod]: because it stands on the official go website i followed their "tutorial" because I didn't work with modules yet

Comment (Al2Klimov) [reply go.mod]: OK... wherever you read example.com, 192.0.2., 2001:db8::, ... they're placeholders.

What do you think you should replace that placeholder with?

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/make-the-project-a-go-module-27`

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**